frontends/web/src/app.tsx

```diff
@@ -149,13 +147,13 @@ class App extends Component<Props, State> {
       route('/', true);
       return;
     }
-    // if on an account that isnt registered route to /
+    // if on an account that isn't registered route to /
     if (inAccounts && !accounts.some(account => currentURL.startsWith('/account/' + account.code))) {
       route('/', true);
       return;
     }
-    // if on index page and we go from zero accounts to at least 1 account, route to /account-summary
-    if (isIndex && oldAccountsLen === 0 && accounts.length) {
+    // if on index page and have at least 1 account, route to /account-summary
+    if (isIndex && accounts.length) {
```

Comment:
@thisconnect this change looks a little risky :P we should test with the routing test table to see if this breaks something else

Reply:
I'm on it, fyi the routing tests only cover none-watchonly situations.